tests/pdgid/test_functions.py

```diff
@@ -36,6 +36,8 @@
 from particle.pdgid import A
 from particle.pdgid import Z
 
+from ..conftest import PDGIDsEnum
```

Comment:
It seems odd to import from conftest. Maybe you should move that to a general location in the tests folder and import from it in conftest and here?

Reply:
Indeed.

You were too fast. I moved to something more standard. Still not fully happy, but OK.